packages/playground/src/explorer/components/node_details_cards/farm_details_card.vue

```diff
@@ -17,7 +17,11 @@ export default {
   props: {
     node: {
```

Comment:
Why are we passing the node to farm details? What if the farm has more than 1 node like freefarm?

Reply:
This component isn't just for farms, this card is used for the nodes page as well so when it is used for the nodes page, the node prop is passed. When it is used for the farms page the farm prop is passed. Hope this clears it up.